### DISCUSSION TITLE
Feature Request: SSL Support

### DISCUSSION THREAD

**Discussion by JesNoFear** [OctoFarm/OctoFarm#1306] [Feature Request]
SSL support and generate your own certificate

Almost everything communicates over SSL anymore, it should at least be an option.

me personally, i use real SSL certs on my internal network, and i know that isnt for everyone, but its real simple to generate a self-signed certificate.

**Comment by NotExpectedYet** [comment]
I'm happy to do this but currently I'm unsure how best to implement it. 

I've always used reverse proxy for SSL termination. 

I'll have to take some time to research it properly but I will look into it. Happy for a PR from someone though.
